manageiq-operator/deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: manageiq.org/v1alpha1
+kind: Manageiq
+metadata:
+  name: example-manageiq
+spec:
+  # Add fields here
+  size: 3
```

Comment:
What is size here? Can we either make this a correct example or remove it?

Reply:
Sure